manifests/claudie/dynamo/job.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-table-job
+  labels:
+    app: dynamo-create-table-job
+spec:
+  template:
+    metadata:
+      labels:
+        app: dynamo-create-table-job
+    spec:
+      restartPolicy: OnFailure
+      volumes:
+        - name: minio-configuration
+          projected:
+            sources:
+              - configMap:
+                  name: minio-cm
+      containers:
+        - name: awc-cli
+          image: amazon/aws-cli
+          args:
+            [
+              "dynamodb",
+              "create-table",
+              "--table-name",
+              "claudie",
+              "--attribute-definitions",
+              "AttributeName=LockID,AttributeType=S",
+              "--key-schema",
+              "AttributeName=LockID,KeyType=HASH",
+              "--provisioned-throughput",
+              "ReadCapacityUnits=1,WriteCapacityUnits=1",
+              "--endpoint-url",
+              "http://dynamo:8000",
+              "--output",
+              "json",
+              "--region",
+              "local",
+            ]
+          env:
+            - name: DYNAMO_ENDPOINT
+              valueFrom:
+                configMapKeyRef:
+                  name: env
+                  key: DYNAMO_HOSTNAME
+            - name: DYNAMO_PORT
+              valueFrom:
+                configMapKeyRef:
+                  name: env
+                  key: DYNAMO_PORT
+            - name: AWS_ACCESS_KEY_ID
+              value: fake
```

Comment:
How is the authentication handled? Are the passwords `fake`, or is there no auth at all?

I'm thinking we could do this similarly as for mongodb.
I.e. kustomize secretgenerator from a file, which contains some `changeme` passwords, which we encourage users to change as they like.

Reply:
AWS cli needs some cred to run the create table command, but since its a locally deployed dynamodb, it does not have any auth. So i need to pass some random value to aws cli for access key and secret key using env vars just to pass the validation check.

Reply:
ok, sounds fine